nessie/lib/queries.py

```diff
@@ -84,6 +84,16 @@ def student_schema():
     return app.config['REDSHIFT_SCHEMA_EDL'] if feature_flag_edl() else app.config['REDSHIFT_SCHEMA_STUDENT']
 
 
+def student_schema_table(key):
+    use_edl = feature_flag_edl()
+    return {
+        'degree_progress': 'student_degree_progress' if use_edl else 'sis_api_degree_progress',
+        'sis_profiles': 'sis_profiles' if use_edl else ('sis_api_profiles_v1' if app.config['STUDENT_V1_API_PREFERRED'] else 'sis_api_profiles'),
```

Comment:
@lyttam @pkerschen - What is our strategy for `sis_profiles` in EDL cutover?

Reply:
@johncrossman To be determined. I think we'll need a job like GenerateMergedStudentFeeds to merge together the individual parts of the profile that we've each been working on along with Canvas data.